service/src/main/java/bio/terra/pipelines/app/configuration/internal/PipelineConfigurations.java

```diff
@@ -0,0 +1,67 @@
+package bio.terra.pipelines.app.configuration.internal;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Map;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@Setter
+@Getter
+@ConfigurationProperties(prefix = "pipelines.configurations")
+public class PipelineConfigurations {
+
+  private PipelinesCommonConfiguration common;
+  private Map<String, ImputationConfig> arrayImputation;
```

Comment:
can you add a comment here to explain that the reason this is a Map<String, ImputationConfig> is because there's a config per version? i.e. the keys in this map represent the versions of the pipeline. (I had to go look at how this was used to figure that out)